plugins/node/opentelemetry-plugin-mongoose/src/mongoose.ts

```diff
@@ -0,0 +1,279 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { BasePlugin } from '@opentelemetry/core';
+import * as shimmer from 'shimmer';
+import * as mongoose from 'mongoose';
+
+import { AttributeNames } from './enums';
+
+import {
+  startSpan,
+  handleError,
+  setErrorStatus,
+  safeStringify,
+  getAttributesFromCollection,
+  handleExecResponse,
+} from './utils';
+
+import { VERSION } from './version';
+
+const contextCaptureFunctions = [
+  'remove',
+  'deleteOne',
+  'deleteMany',
+  'find',
+  'findOne',
+  'estimatedDocumentCount',
+  'countDocuments',
+  'count',
+  'distinct',
+  'where',
+  '$where',
+  'findOneAndUpdate',
+  'findOneAndDelete',
+  'findOneAndReplace',
+  'findOneAndRemove',
+];
+
+// when mongoose functions are called, we store the original call context
+// and then set it as the parent for the spans created by Query/Aggregate exec()
+// calls. this bypass the unlinked spans issue on thenables await operations (issue #29)
+export const _STORED_PARENT_SPAN: unique symbol = Symbol('stored-parent-span');
```

Comment:
Currently exporting here means end-user could have access to the symbol, i think you should move it inside the utils and importing it here